src/routes/reference/basic-reactivity/create-signal.mdx

```diff
@@ -9,153 +9,117 @@ tags:
-	```
+#### `name`
 
-:::
+- **Type:** `string`
+- **Default:** `undefined`
 
-## Options
+A name for the signal used for debugging purposes in development mode.
+The name will show up in console messages and in the [Solid devtools](https://github.com/thetarnav/solid-devtools).
 
-| Name       | Type                                       | Default | Description                                                                                                                                                                                                                                                         |
-| ---------- | ------------------------------------------ | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `equals`   | `false \| ((prev: T, next: T) => boolean)` | `===`   | A function that determines whether the Signal's value has changed. If the function returns true, the Signal's value will not be updated and dependents will not rerun. If the function returns false, the Signal's value will be updated and dependents will rerun. |
-| `name`     | `string`                                   |         | A name for the Signal. This is useful for debugging.                                                                                                                                                                                                                |
-| `internal` | `boolean`                                  | `false` | If true, the Signal will not be accessible in the devtools.                                                                                                                                                                                                         |
+#### `equals`
 
-### `equals`
+- **Type:** `false | ((prev: T, next: T) => boolean)`
+- **Default:** `false`
 
-The `equals` option can be used to customize the equality check used to determine whether the Signal's value has changed.
-By default, the equality check is a strict equality check (`===`).
-If you want to use a different equality check, you can pass a custom function as the `equals` option.
-The custom function will be called with the previous and next values of the Signal as arguments.
-If the function returns true, the Signal's value will not be updated and dependents will not rerun.
-If the function returns false, the Signal's value will be updated and dependents will rerun.
+By default, signals use reference equality (`===`).
+A custom comparison function to determine when the signal should update.
+If set to `false`, the signal will always update regardless of value equality.
+This can be useful to create a Signal that triggers manual updates in the reactive system, but must remain a pure function.
```

Comment:
```suggestion
A custom comparison function to determine whether the signal should update.
If set to `false`, the signal will always update regardless of value equality.
It must be a pure function.
By default, the strict equality operator (`===`) is used.
```

I think it's better to first explain its functionality, then the default behavior.  Also, I found the example contradictory. How can it be a pure function while also updating other signals?